tests/unit/test_storage_db.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `ebr_trackerbot` package."""
+
+import sys
+import os
+
+sys.path.append("ebr_trackerbot")
+sys.path.append("ebr_trackerbot/storage")
+
+import logging
+import db
+from unittest.mock import Mock
+
+
+def test_storage_db_create_table():
+    conn = Mock()
+    executeMock = Mock()
+    conn.cursor.return_value = executeMock
+
+    db.create_table(lambda: conn)
```

Comment:
I don't think this test is necessary as it is testing the original Python module itself which is (or should be) already tested

Reply:
This unit test checks that custom function create_table calls specific sql command